src/client.js

```diff
@@ -316,7 +316,11 @@ Client.prototype.updateSession = function(token, ip, request) {
 	}
 
 	if (agent.os.name) {
-		friendlyAgent += ` on ${agent.os.name} ${agent.os.version}`;
+		friendlyAgent += ` on ${agent.os.name}`;
+	}
+
+	if (agent.os.version) {
```

Comment:
Should this be inside of the previous `if`?

Reply:
Oh, probably.

Coding at 25 hours awake problems... I'll go to sleep right after fixing that. :sleeping: